src/main/kotlin/com/hm/hyeonminshinlottospring/domain/winninglotto/domain/WinningLotto.kt

```diff
@@ -0,0 +1,36 @@
+package com.hm.hyeonminshinlottospring.domain.winninglotto.domain
+
+import com.hm.hyeonminshinlottospring.domain.lotto.domain.Lotto
+import jakarta.persistence.Column
+import jakarta.persistence.Entity
+import jakarta.persistence.GeneratedValue
+import jakarta.persistence.GenerationType
+import jakarta.persistence.Id
+import jakarta.persistence.Index
+import jakarta.persistence.JoinColumn
+import jakarta.persistence.OneToOne
+import jakarta.persistence.SequenceGenerator
+import jakarta.persistence.Table
+
+@Table(
+    indexes = [
+        Index(name = "lotto_id_index", columnList = "lotto_id"),
```

Comment:
아하 FK는 보통 명시적으로 인덱스라고 써두지 않아도 좋아요..!
대신 Oracle DB는 제외구요 ㅋㅋㅋ..
오라클 디비는 FK도 인덱스가 필요해요

Reply:
```suggestion
        Index(name = "lotto_id", columnList = "lotto_id"),
```
그럼 이렇게 잡는게 오라클 디비 제외 일반적인 형태라고 할 수 있나요?

Reply:
보통 DB가 알아서 걸어주는 FK인덱스를 제외한 개발자들이 명시적으로 넣어주는 인덱스를 적어둡니다~!
따라서 FK의 인덱스는 잘 안적어주는걸 선호해요!
오라클은 FK인덱스도 적어주는게 좋구요 ㅋㅋㅋ

Reply:
아하 그럼 FK를 제외한 인덱스의 경우에만 적어주는 것이 관례라는 것이군요!
오늘도 하나 알아가네요 ㅎㅎ